ChatSecure/Classes/Controllers/XMPP/OTRXMPPRoomManager.m

```diff
@@ -308,23 +329,7 @@ - (void)xmppRoomDidJoin:(XMPPRoom *)sender
         
         //Invite buddies
         NSArray<NSString*> *buddyUniqueIds = [self.inviteDictionary objectForKey:sender.roomJID.bare];
-        if (!buddyUniqueIds.count) {
```

Comment:
this probably belongs here
```objc
[self.inviteDictionary removeObjectForKey:room.roomJID.bare];
```

Reply:
Oops, you're right, that should have remained where it was.